include/alpaka/mem/view/Traits.hpp

```diff
@@ -353,4 +376,104 @@ namespace alpaka
     {
         return traits::CreateStaticDevMemView<TDev>::createStaticDevMemView(pMem, dev, extent);
     }
+
+    //! Creates a view to a device pointer
+    //!
+    //! \param dev Device from where pMem can be accessed.
+    //! \param pMem Pointer to memory. The pointer must be accessible from the given device.
+    //! \param extent Number of elements represented by the pMem.
+    //!               Using a multi dimensional extent will result in a multi dimension view to the memory represented
+    //!               by pMem.
+    //! \return A view to device memory.
+    template<typename TDev, typename TElem, typename TExtent>
+    auto createView(TDev const& dev, TElem* pMem, TExtent const& extent)
+    {
+        using Dim = alpaka::Dim<TExtent>;
+        using Idx = alpaka::Idx<TExtent>;
+        auto const extentVec = Vec<Dim, Idx>(extent);
+        return traits::CreateViewPlainPtr<TDev>::createViewPlainPtr(
+            dev,
+            pMem,
+            extentVec,
+            detail::calculatePitchesFromExtents<TElem>(extentVec));
+    }
+
+    //! Creates a view to a device pointer
+    //!
+    //! \param dev Device from where pMem can be accessed.
+    //! \param pMem Pointer to memory. The pointer must be accessible from the given device.
+    //! \param extent Number of elements represented by the pMem.
+    //!               Using a multi dimensional extent will result in a multi dimension view to the memory represented
+    //!               by pMem.
+    //! \param pitch Pitch in bytes for each dimension. Dimensionality must be equal to extent.
+    //! \return A view to device memory.
+    template<typename TDev, typename TElem, typename TExtent, typename TPitch>
+    auto createView(TDev const& dev, TElem* pMem, TExtent const& extent, TPitch const& pitch)
+    {
+        return traits::CreateViewPlainPtr<TDev>::createViewPlainPtr(dev, pMem, extent, pitch);
+    }
+
+    //! Creates a view to a device std::vector
+    //!
+    //! \param dev Device from where vec can be accessed.
+    //! \param vec std::vector container. The data hold by the container the must be accessible from the given device.
+    //! \return A view to device memory.
+    template<typename TDev, typename TElem, typename TAllocator>
+    auto createView(TDev const& dev, std::vector<TElem, TAllocator>& vec)
+    {
+        //! \todo With C++17 uste std::data() and add support for all contagious container
```

Comment:
Lol, `contagious` containers with zombies.
Should be `contiguous`.

Reply:
Also, typo in `use`

Reply:
> Lol, contagious containers with zombies.

Maybe I read to much Covid news ^^

I fixed the typo